amps/ags/rm-automation/rm-automation-community-rest-api/src/main/java/org/alfresco/rest/rm/community/requests/gscore/GSCoreAPI.java

```diff
@@ -193,4 +194,6 @@ public ActionsExecutionAPI usingActionsExecutionsAPI()
     }
 
     public HoldsAPI usingHoldsAPI() { return new HoldsAPI(getRmRestWrapper()); }
+
+    public RetentionScheduleAPI usingRetentionScheduleAPI() { return new RetentionScheduleAPI(getRmRestWrapper()); }
```

Comment:
formatting issue for opening and closing bracket

Reply:
done the changes as per the comment.